.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,12 +4,11 @@ relates to OR closes: #XXXX
 
 ---
 
-Before we can merge this PR, please make sure that all the following items have been
-checked off. If any of the checklist items are not applicable, please leave them but
-write a little note why.
-
+Before we can approve this PR for merge, please make sure that **all** the following items have been checked off:
 - [ ] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
-- [ ] Added **one** line describing your change in `<branch>/CHANGELOG.md`
+- [ ] Added *exactly* one label from the `L-` group against this PR
+- [ ] Added *at least* one label from the `A-` and one label from the `C-` group against either this PR or its associated issue
```

Comment:
This `either this PR or its associated issue` stuff is clumsy.

I think we need to figure out a way to resolve it so all the info ( all labels and the milestone ) is always in the PR ( except for the problem description, that may be in the linked issue)

Looking for ideas @ghzlatarev @Dengjianping @Apokalip @flame4

Reply:
How can PR be categorized? I suppose some PRs are bug fix, some are new features, and some are improvements.

Reply:
Maybe we need to introduce more labels like:

`Added`: for new features.
`Changed`:  for changes in existing functionality.
`Deprecated`: for soon-to-be removed features.
`Removed`: for now removed features.
`Fixed`: for any bug fixes.
`Security`: fix a security issue.

Reply:
I think @Apokalip is doing that in his #661 PR

> Maybe we need to introduce more labels like:
> 
> `Added`: for new features. `Changed`: for changes in existing functionality. `Deprecated`: for soon-to-be removed features. `Removed`: for now removed features. `Fixed`: for any bug fixes. `Security`: fix a security issue. `Dolphin` `Calamari` `Manta`